packages/neo-one-node-core/src/Blockchain.ts

```diff
@@ -149,7 +150,7 @@ export interface Blockchain extends BlockchainStorage {
 
   readonly getValidators: (transactions: readonly Transaction[]) => Promise<readonly ECPoint[]>;
 
-  readonly invokeScript: (script: Buffer) => Promise<CallReceipt>;
+  readonly invokeScript: (script: Buffer, signers: readonly Signer[]) => Promise<CallReceipt>;
   readonly invokeTransaction: (transaction: InvocationTransaction) => Promise<CallReceipt>;
   readonly calculateClaimAmount: (inputs: readonly Input[]) => Promise<BN>;
```

Comment:
we should get on the same page with `node-core` stuff. Because the plan was to not touch any of the packages now.